utils/production-automation/run-amplicon.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env /hps/nobackup2/production/metagenomics/pipeline/tools-v5/miniconda3-4.6.14/bin/python3
+import logging
+import os
+import re
+import yaml
+import argparse
+from subprocess import CalledProcessError
+from subprocess import check_output
+
+cwlSingle = '/hps/nobackup2/production/metagenomics/pipeline/tools-v5/pipeline-v5/workflows/amplicon-wf-single-empty.cwl'
+cwlPaired = '/hps/nobackup2/production/metagenomics/pipeline/tools-v5/pipeline-v5/workflows/amplicon-wf-paired-empty.cwl'
+ymlSingle = '/hps/nobackup2/production/metagenomics/pipeline/tools-v5/pipeline-v5/workflows/amplicon-wf-single-job.yml'
+ymlPaired = '/hps/nobackup2/production/metagenomics/pipeline/tools-v5/pipeline-v5/workflows/amplicon-wf-paired-job.yml'
+toil_mem= '30G'
+amplicon_mem= 20000
+stats_mem = 1000
+restart_mem= 10000
+toil_cores = 4.0
+lsf_cores = 4
+groupname = '/toil-test'
+queue = 'production-rh74'
+
+def setup_dir(projectID, pathtoProject):
```

Comment:
General feedback: I would be try to be consistent with the naming conventions (lower_case_with_underscores). this also applies for method parameters.
https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions